docs/scripts/formattedTSDemos.js

```diff
@@ -106,14 +106,14 @@ async function transpileFile(tsxPath, program) {
       },
     });
     const codeWithPropTypes = typescriptToProptypes.inject(propTypesAST, code);
-
     const prettierConfig = prettier.resolveConfig.sync(jsPath, {
       config: path.join(workspaceRoot, 'prettier.config.js'),
     });
     const prettierFormat = (jsSource) =>
       prettier.format(jsSource, { ...prettierConfig, filepath: jsPath });
 
-    const prettified = prettierFormat(codeWithPropTypes);
+    const codeWithoutTsIgnoreComments = codeWithPropTypes.replace(/^\s*\/\/ @ts-ignore.*$/gm, '');
```

Comment:
Should we also remove `ts-expect-error`, or aren't there any occurrences of it?

Reply:
It's about @ts-ignore comments added in the tsx files that are automatically added in the .js files while running `yarn docs:typescript:formatted` - @ts-ignore does not make sense in the JavaScript files.